tests/testcore/functional_test_base.go

```diff
@@ -252,6 +270,7 @@ func (s *FunctionalTestBase) initAssertions() {
 	s.ProtoAssertions = protorequire.New(s.T())
 	s.HistoryRequire = historyrequire.New(s.T())
 	s.UpdateUtils = updateutils.New(s.T())
+	s.checkNoUnexpectedErrorLogs() // should have already been called in TearDownTest, but just in case
```

Comment:
This is not ideal as it labels the wrong test as failed, but it's better than forgetting about it altogether.